src/slack_api_util.ts

```diff
@@ -168,6 +203,64 @@ export function copyUserInfoToDbMessageEntry(
     userInfo.lastVoterMessageSecsFromEpoch;
 }
 
+export async function fetchSlackChannelMap(): Promise<
+  [Record<string, string>, Record<string, string>]
+> {
+  const response = await slackAPI.get('conversations.list', {
+    params: {
+      limit: 1000,
```

Comment:
Slack recommends no more than 200 at a time. We have a function fetchSlackChannelNamesAndIds in this file that already does this with pagination https://api.slack.com/methods/conversations.list